modules/nf-core/fq/generate/meta.yml

```diff
@@ -0,0 +1,40 @@
+name: "fq_generate"
+description: |
+  fq generate is a FASTQ file pair generator. It creates two reads, formatting names as described by Illumina.
+  While generate creates "valid" FASTQ reads, the content of the files are completely random. The sequences do not align to any genome.
```

Comment:
The function requires a random seed. It would be good to make it explicit that this needs to be set.